HeterogeneousCore/CUDATest/test/testMissingDictionaryCUDA.sh

```diff
@@ -0,0 +1,13 @@
+#! /bin/bash
+
+function die {
+  echo -e "$1"
+  echo "exit status $2"
+  exit $2
+}
+
+LOCAL_TEST_DIR=${SCRAM_TEST_PATH:-$CMSSW_BASE/src/HeterogeneousCore/CUDATest/test}
+
+cmsRun ${LOCAL_TEST_DIR}/testMissingDictionaryCUDA_cfg.py >& testMissingDictionaryCUDA.log && die "The cmsRun test job succeeded unexpectedly" $?
```

Comment:
With `&&`, i.e. when the `cmsRun` is expected to fail but it succeeds, the `$?` is `0` and the `die` function above ends up calling `exit 0`, i.e. this script would succeed. I'd suggest to use explicit non-zero exit code in this case, e.g.
```sh
cmsRun ${LOCAL_TEST_DIR}/testMissingDictionaryCUDA_cfg.py >& testMissingDictionaryCUDA.log && die "The cmsRun test job succeeded unexpectedly" 1
```